### PR TITLE
dbscoach/make rules final

### PR DESCRIPTION
**Pull request by dbscoach**
### Please describe your change in one or two sentences

Removed references to rules being a draft so final rules PDF can be rendered.

### Please explain why do you think this change should be in the rules

will "wait and revisit if turns out to become problematic" for #68

**Please consider merging #95 , #96 , #97 into this one before merging this to master.**

### CODE REVIEW

File: preamble.tex [hunk 1]
```diff
@@ -73,17 +74,17 @@
 %\renewcommand{\topmargin}{-20pt}
 
 \rfoot{Page \textbf{\thepage} of \textbf{\pageref{LastPage}}}
-\lfoot{\textit{Final rules as of \today}}
+\lfoot{\textit{Draft rules as of \today}}
```
Comment (mrshu) [inline preamble.tex:77]: @dbscoach if these are to be made final, should this change as well?

**Comment by mrshu** [comment]
Thank you for the massive amount of work you did here @dbscoach.

I had only one small question, otherwise this seems to be ready to go :slightly_smiling_face:

File: preamble.tex (hunk 1 above)
Comment (dbscoach) [reply preamble.tex:77]: Absolutely, my CTRL-F didn't find that...

**Comment by mrshu** [comment]
Thank you for the massive amount of work you did here @dbscoach !

**Review by mrshu — APPROVED** [review]

**Pull request merged**